### PR TITLE
Upgrade mariner images to LLVM 16

### PR DESCRIPTION
**Pull request by sbomer**
Now that all known issues with the LLVM12-based images are fixed we should be able to upgrade them again. @jkoritzinsky I compared the sanitizer static libs available on one of the machines (I picked arm64) and this is the diff:

```diff
> libclang_rt.asan_static-aarch64.a
> libclang_rt.hwasan-preinit-aarch64.a
< libclang_rt.scudo-aarch64.a
< libclang_rt.scudo_cxx-aarch64.a
< libclang_rt.scudo_cxx_minimal-aarch64.a
< libclang_rt.scudo_minimal-aarch64.a
> libclang_rt.scudo_standalone-aarch64.a
> libclang_rt.scudo_standalone_cxx-aarch64.a
```

### CODE REVIEW

File: src/cbl-mariner/2.0/cross/arm/Dockerfile [hunk 1]
```diff
@@ -1,45 +1,46 @@
...
 
-RUN tdnf install -y debootstrap
-
 RUN /scripts/eng/common/cross/build-rootfs.sh arm xenial --skipunmount
 
 # Build compiler-rt profile library for PGO instrumentation
 RUN mkdir compiler-rt_build && cd compiler-rt_build && \
     BUILD_FLAGS="-v --sysroot=$ROOTFS_DIR" \
-    TARGET_TRIPLE=arm-linux-gnueabihf && \
+    TARGET_TRIPLE="arm-linux-gnueabihf" && \
```
Comment (janvorli) [inline src/cbl-mariner/2.0/cross/arm/Dockerfile:9]: I'm just curious - why weren't the double quotes necessary before?

**Review by janvorli — APPROVED** [review]
LGTM, thank you!

File: src/cbl-mariner/2.0/cross/arm/Dockerfile (hunk 1 above)
Comment (janvorli) [reply src/cbl-mariner/2.0/cross/arm/Dockerfile:9]: Ok, makes sense.

**Review by jkoritzinsky — APPROVED** [review]
Sanitizer file changes make sense. LGTM

**Pull request merged**